src/App.tsx

```diff
@@ -42,20 +42,19 @@ import {CLOUD} from 'src/shared/constants'
 import {executeVWO} from 'src/utils/vwo'
 
 // Styles
-import './MultiOrgOverrideStyles.scss'
+import 'src/identity/styles/MultiOrgStyles.scss'
 
```

Comment:
I don't think these styles should live in the root directory. I'd prefer keeping them with the other user-identity-related components.